spec/api/v2/schemas/report.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require './spec/api/v2/schemas/util'
+
+module Api
+  module V2
+    module Schemas
+      # :nodoc:
+      module Report
+        extend Api::V2::Schemas::Util
+
+        REPORT = {
+          type: :object,
+          required: %w[compliance_threshold profile_id],
```

Comment:
Does this even make sense here?

Reply:
No, since the attributes are read only. Removing.